hrjobcontract/js/src/job-contract/services/contract-leave.js

```diff
@@ -155,6 +155,25 @@ define([
                         return null;
                     }
 
+                    /**
+                     * This is a little helper method to calculate the estimated leave amount for a
+                     * leave type.
+                     *
+                     * If this leave type has "add_public_holidays" set, the estimated amount will be
+                     * the leave amount + the given number of Public Holidays. Otherwise, the estimated
+                     * amount is the same as the leave amount.
+                     *
+                     * @param {number} numberOfPublicHolidays
+                     * @returns {number}
+                     */
+                    modelEntry.getEstimatedLeaveAmount = function(numberOfPublicHolidays) {
+                      if(this.add_public_holidays == '1') {
+                        return parseFloat(this.leave_amount) + parseFloat(numberOfPublicHolidays);
```

Comment:
Out of curiosity, what values are you expecting to come from the api? (given the usage of `parseFloat')